systems/identity/identity-adapter-out/src/main/kotlin/hs/kr/entrydsm/identity/adapterout/pass/KcbLicenseFileInitializer.kt

```
@@ -0,0 +1,31 @@
+    private val licenseFilePath: String,
+) {
+    @PostConstruct
+    fun initialize() {
+        if (licenseFileUrl.isBlank()) return
+        val target = Path.of(licenseFilePath)
+        try {
+            target.parent?.let(Files::createDirectories)
+            URI(licenseFileUrl).toURL().openStream().use { input ->
+                Files.copy(input, target, StandardCopyOption.REPLACE_EXISTING)
+            }
+        } catch (exception: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)